### ISSUE TITLE
Console and Exec content does not use whole available window size

### ISSUE THREAD

**Issue by valkuc** [lxdware/lxd-dashboard#64]
It would be nice if xterm content will occupy all available width.
![изображение](https://user-images.githubusercontent.com/6421082/212540233-7e6ffdae-98fb-44fd-8916-42132cf1a9e1.png)

**Comment by matthewalanpenning** [comment]
I agree. When I wrote the code I had tried to get this to work but could quite get it. I am looking to rewrite this and will work again to try to get the resize working.

**Comment by valkuc** [comment]
That would be great, as "Console" and "Exec" features are quite useful!
